docs/tuning-guide.md

```diff
@@ -337,7 +337,7 @@ Custom Spark SQL Metrics are available which can help identify performance bottl
 
 Not all metrics are enabled by default. The configuration setting `spark.rapids.sql.metrics.level` can be set
 to `DEBUG`, `MODERATE`, or `ESSENTIAL`, with `MODERATE` being the default value. More information about this
-configuration option is available in the <a href="configs.md#sql.metrics.level">configuration</a> documentation.
+configuration option is available in the [`configuration`](configs.md#sql.metrics.level) documentation.
```

Comment:
Backticks are usually reserved for cases where we are referring to code or things the user should type verbatim.  This isn't one of those cases.
```suggestion
configuration option is available in the [configuration documentation](configs.md#sql.metrics.level).
```